static_src/components/app_container.jsx

```diff
@@ -133,8 +136,22 @@ export default class AppContainer extends React.Component {
           <Panel title="Usage and allocation">
             <UsageLimits app={ this.state.app } quota={ this.state.quota } />
           </Panel>
-          <RoutesPanel />
-          <ServiceInstancePanel />
+
+          <PanelGroup>
+            <PanelGroup columns={ 6 }>
+              <div className={ this.styler('panel-routes')}>
```

Comment:
This block could use some tab formatting, I'm a little unsure of what's going on.